packages/admin/src/pages/build-application/[applicationId]/[sectionId]/question-type/add-word-count.page.tsx

```diff
@@ -32,40 +37,94 @@ export const getServerSideProps = async (
       (section) => section.sectionId === sectionId
     ) as ApplicationFormSection;
 
+    const queryString = context.query
+      ? `?${new URLSearchParams({ ...(context.query as object) })}`
```

Comment:
redundant object spread here

```suggestion
      ? `?${new URLSearchParams(context.query as object)}`
```